average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+   const newArr = numbers.filter( value => !Number.isNaN(value) );
```

Comment:
The idea of naming variables is to let readers understand their intent without having to read the entire code.
Try another name instead of `newArr` to express its intent.